pt-BR/mail/chrome/messenger-newsblog/am-newsblog.dtd

```diff
@@ -1,22 +1,22 @@
-<!ENTITY newsblogTitle.label "RSS Account Settings">
+<!ENTITY newsblogTitle.label "Configurações de Conta RSS">
 
-<!ENTITY indexMsgBodies.label "Index message bodies">
+<!ENTITY indexMsgBodies.label "Indexar corpo das mensagens">
 <!ENTITY indexMsgBodies.accesskey "I">
 
 <!ENTITY biffStart.label "Verificar novos itens a cada ">
 <!ENTITY biffStart.accesskey "V">
 <!ENTITY biffEnd.label "minutos">
 
-<!ENTITY loginAtStartup.label "Verificar novos itens ao iniciar"> 
+<!ENTITY loginAtStartup.label "Verificar novos itens ao iniciar">
 <!ENTITY loginAtStartup.accesskey "e">
 
-<!ENTITY useQuickMode.label "Por padrão, exibir o conteúdo do item em vez de carregar a página web"> 
+<!ENTITY useQuickMode.label "Por padrão, exibir o conteúdo do item em vez de carregar a página web">
 <!ENTITY useQuickMode.accesskey "P">
 
-<!ENTITY manageSubscriptions.label "Inscrições RSS…"> 
+<!ENTITY manageSubscriptions.label "Inscrições RSS…">
```

Comment:
Paulo, this wasn't translated by me, and it's already this way on the current Postbox version. To avoid clients' confusion, I believe it's better not to change it.
Please note that the file showed up because Atom automatically removed the space in the end of line 16.